things/model/src/main/java/org/eclipse/ditto/things/model/ImmutableFeatureProperties.java

```diff
@@ -212,11 +212,21 @@ public boolean contains(final CharSequence key) {
         return wrapped.contains(key);
     }
 
+    @Override
+    public boolean containsFlatteningArrays(final CharSequence key) {
+        return wrapped.contains(key);
```

Comment:
Shouldn’t this call "containsFlatteningArrays" on the wrapped object?

Reply:
@alstanchev very good catch - stupid copy&paste error :/
Will fix, thanks a lot